src/Sulu/Bundle/AdminBundle/Resources/js/components/Navigation/Navigation.js

```diff
@@ -141,10 +153,21 @@ export default class Navigation extends React.Component<Props> {
                 <div className={navigationStyles.items}>
                     {this.cloneChildren()}
                 </div>
-                <div className={navigationStyles.versions}>
-                    {this.renderAppVersion()}
-                    <div>
-                        Sulu (<a href={suluVersionLink} rel="noopener noreferrer" target="_blank">{suluVersion}</a>)
+                <div className={navigationStyles.footer}>
+                    {onPinClick &&
+                        <div className={navigationStyles.pinContainer}>
+                            <ButtonGroup>
+                                <Button className={navigationStyles.pin} onClick={this.handlePinClick}>
```

Comment:
I would have imagined this button to stay active - i.e. same style as on hovered - when the navigation is pinned, so that it is easy recognizable if the navigation is pinned or not.

Reply:
And is it possible that the button in the design is quite a bit smaller?